src/components/views/auth/CaptchaForm.tsx

```diff
@@ -103,8 +103,8 @@ export default class CaptchaForm extends React.Component<ICaptchaFormProps, ICap
     }
 
     private resetRecaptcha() {
-        if (this.captchaWidgetId !== null) {
-            global.grecaptcha.reset(this.captchaWidgetId);
+        if (this.captchaWidgetId) {
+            global?.grecaptcha?.reset(this.captchaWidgetId);
```

Comment:
also, we tend to use `window` instead of global outside of the js-sdk layer

Reply:
@t3chguy it's `global` in the entire file, so I went on with that.